GVFS/GVFS.Platform.Windows/packages.config

```diff
@@ -4,6 +4,7 @@
   <package id="Microsoft.Database.Collections.Generic" version="1.9.4" targetFramework="net461" />
   <package id="Microsoft.Database.Isam" version="1.9.4" targetFramework="net461" />
   <package id="Microsoft.Windows.ProjFS" version="1.0.19108.2" targetFramework="net461" />
+  <package id="GVFS.ProjFS" version="2019.411.1" targetFramework="net461" />

```

Comment:
@wilbaker - I would be curious if there is a reason to not include this package as a dependency of `GVFS.Platform.Windows`...

Reply:
> I would be curious if there is a reason to not include this package as a dependency of GVFS.Platform.Windows

It's a bit weird but strictly speaking `GVFS.Platform.Windows` only has a dependency on the `Microsoft.Windows.ProjFS` package.

The only thing that *really* needs the `GVFS.ProjFS` package is the installer because the package is only needed if ProjFS is not available as part of the operating system.

Was there a reason you had to add this?

Reply:
A bit more history here, `GVFS.Platform.Windows` did have a reference to `GVFS.ProjFS` when the managed API and native libraries were in the same package.  When they were split into two packages we removed the reference to `GVFS.ProjFS`.

Unless this was required to fix the build I recommend taking the "GVFS.ProjFS" reference back out, as `GVFS.Platform.Windows` should have to care about the native library.

Reply:
> The only thing that really needs the GVFS.ProjFS package is the installer because the package is only needed if ProjFS is not available as part of the operating system.

What seems awkward is that the the `ProjectedFSLib.dll` is copied into the `GVFS.Platform.Windows` project's output directory, and that the installer copies it out of that directory.

When would we build `GVFS.Platform.Windows` and not want to copy `ProjectedFSLib.dll` into the output directory. Is there a scenario where this actually happens today?